lib/models/device.coffee

```diff
@@ -509,6 +544,76 @@ exports.move = (uuid, application, callback) ->
 	.nodeify(callback)
 
 ###*
+# @summary Start application on device
+# @name start
+# @public
+# @function
+# @memberof resin.models.device
+#
+# @param {String} uuid - device uuid
+# @fulfil {String} - application container id
+# @returns {Promise}
+#
+# @example
+# resin.models.device.start('7cf02a6').then(function(containerId) {
+# 	console.log(containerId);
+# });
+#
+# @example
+# resin.models.device.start('7cf02a6', function(error, containerId) {
+# 	if (error) throw error;
+# 	console.log(containerId);
+# });
+###
+exports.start = (uuid, callback) ->
+	exports.get(uuid).then (device) ->
+		appId = device.application[0].id
+		return request.send
+			method: 'GET'
+			url: "/supervisor/v1/apps/#{appId}/start"
+			baseUrl: settings.get('apiUrl')
+			body:
+				deviceId: device.id
+				appId: appId
+	.get('body')
+	.nodeify(callback)
+
+###*
+# @summary Stop application on device
+# @name stop
+# @public
+# @function
+# @memberof resin.models.device
+#
+# @param {String} uuid - device uuid
+# @fulfil {String} - application container id
```

Comment:
Same as above. This function resolves and object containing a `containerId` property, but not the property itself.